bindings/python/pyiree/rt/CMakeLists.txt

```diff
@@ -12,77 +12,59 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-iree_py_library(
-  NAME
-    rt
+iree_add_pyext_library(IREEBindingsPythonRtLib
   SRCS
-    "__init__.py"
-    # "system_api.py"
-  DEPS
-    ::binding
-)
-
-iree_py_extension(
-  NAME
-    binding
-  SRCS
-    "initialize_module.cc"
-  DEPS
-    iree::hal::vulkan::vulkan_driver_module
-    iree::hal::llvmjit::llvmjit_driver_module
-    iree::hal::vmla::vmla_driver_module
-    ::rt_library
-    bindings::python::pyiree::common
-    iree::base::initializer
-  COPTS
-    ${PYBIND_COPTS}
-    ${PYBIND_EXTENSION_COPTS}
-  TYPE
-    SHARED
-)
-
-iree_pybind_cc_library(
-  NAME
-    rt_library
-  HDRS
     "function_abi.h"
     "hal.h"
     "host_types.h"
     "vm.h"
-  SRCS
     "function_abi.cc"
     "hal.cc"
     "host_types.cc"
     "vm.cc"
+  PYEXT_DEPS
+    IREEBindingsPythonCommonLib
   DEPS
-    bindings::python::pyiree::common
-    iree::base::api
-    iree::base::signature_mangle
-    iree::hal::api
-    iree::modules::hal
-    iree::modules::strings::strings_module
-    iree::modules::tensorlist::native_module
-    iree::vm
-    iree::vm::bytecode_module
-    iree::vm::invocation
-    iree::vm::ref
-    iree::vm::variant_list
-    absl::inlined_vector
-    absl::memory
-    absl::strings
-    absl::optional
-    absl::span
-  TYPE
-    STATIC
+    iree_base_api
```

Comment:
you should still be able to use the aliases - I'd strongly prefer that so that we only have one way of writing the name of something and find/replace/etc work